# K erros
alpha1: 0.05
alpha2: 0.05
alpha3: 0.05
alpha4: 0.05
